File: app/src/main/java/com/mouredev/weeklychallenge2022/Challenge6.js
const invertirCadena = (cadena) =>{
let cadenaInvertida = "";
for(let i= cadena.length-1; i >=0; i--){
  cadenaInvertida += cadena[i];
}
console.log(cadenaInvertida);
}

const texto = "Hola Mundo";

invertirCadena(texto);

File: app/src/main/java/com/mouredev/weeklychallenge2022/Challenge6.py
"""/*
 * Crea un programa que invierta el orden de una cadena de texto
 * sin usar funciones propias del lenguaje que lo hagan de forma automática.
 * - Si le pasamos "Hola mundo" nos retornaría "odnum aloH"
 */"""
 
def invierte(cadena):
    cadenainvertida = ""
    for caracter in cadena:
        cadenainvertida = caracter + cadenainvertida
    print (cadenainvertida)

cadena = input("Dime una cadena: ")
invierte(cadena)
